index.bs

```diff
@@ -150,7 +196,20 @@ the following steps:
 1. Return |encodedOutput|.
 
 <h3 algorithm id="delivery-time">Establishing report delivery time</h3>
-TODO
+To <dfn>establish report delivery time</dfn> given an [=impression=] and a
+[=conversion/conversion time=], run the following steps:
```

Comment:
I might write

```suggestion
The <dfn>report delivery time</dfn> for an [=impression=] |impression| and a
[=conversion/conversion time=] |conversionTime| is the result of the following steps:
```
That phrasing tends to be easier to call. I wonder if "report delivery time" is going to be ambiguous with other reporting APIs, which would argue for "conversion report delivery time".

You might also have this take the conversion instead of just the conversion time, to make the calls a bit simpler and to give the arguments more semantic types.